features/step_definitions/networking.rb

```diff
@@ -533,16 +533,22 @@
   _node = node
 
   restart_network = proc {
-    # For 3.10 version, should delete the sdn pod to restart network components
-    if env.version_ge("3.10", user: user)
-      logger.info("OCP version >= 3.10")
-      sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: _admin) { |pod, hash|
-        pod.node_name == _node.name
-      }.first
-      @result = sdn_pod.ensure_deleted(user: _admin)
-    else
-      step "the node service is restarted on the host"
-    end
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  raise "Unable to find corresponding networkType pod name" unless @result[:success]
+  if @result[:response] == "OpenShiftSDN"
+     app="app=sdn"
+     project_name="openshift-sdn"
+  else
+     app="app=ovnkube-node"
+     project_name="openshift-ovn-kubernetes"
+  end
+  sdn_pod = BushSlicer::Pod.get_labeled(app, project: project(project_name, switch: false), user: admin) { |pod, hash|
+    pod.node_name == node.name
+  }.first
+  @result = sdn_pod.ensure_deleted(user: _admin)
+  unless @result[:success]
+    raise "Fail to delete sdn/ovn pod"
+  end
```

Comment:
I think we need to change all these to support host OVS in 4.6

You can see the WIP in #1252 

https://github.com/openshift/verification-tests/pull/1252/files#diff-b4003f8a21fcd4d7140fd9ca8d001ac7R620

I'm also trying to add a BushSclier::NetworkOperator to get the network_type

https://github.com/openshift/verification-tests/pull/1252/files#diff-133bb191d9c8a457a4a8dffc144cd707R9

```ruby
      network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
      network_type = network_operator.network_type(_admin)
      case network_type
      when "OpenShiftSDN"
        net_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: _admin) { |pod, hash|
          pod.node_name == _node.name
        }.first
      when "OVNKubernetes"
        net_pod = BushSlicer::Pod.get_labeled("app=ovnkube-node", project: project("openshift-ovn-kubernetes", switch: false), user: admin) { |pod, hash|
          pod.node_name == _node.name
        }.first
      else
        raise "unknown network_type"
      end
```

Reply:
yea, I agree. We need gating criteria in #1252 for 4.6+ vs <4.6. Lets keep this PR open for a while until 1252 merges. This might eventually have to be closed. Thanks